articles/best-practices/application-settings.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Recommended application settings in Auth0.
+topics:
+  - best-practices
+  - configuration
+  - application
+  - settings
+contentType: 
+    - index
+    - reference
+useCase:
+  - best-practices
+  - application
+  - application-settings
+---
+
+# Recommended Application Settings
+
+In this article, you'll learn some best practices for configuring [applications](/applications).
+
+## Check the Client ID
+
+Confirm your application code uses the correct Client ID. You can find the Client ID in your [application's settings](${manage_url}/#/applications) on the Auth0 Dashboard.
+
+## Correct application type
+
+Make sure the correct [application type](/applications/application-types) is set in your application settings. Setting the correct application type helps Auth0 check for certain security risks.
+
+## Flag third party applications
+
+You should flag [first-party and third-party applications](/applications/application-types#first-vs-third-party-applications). First-party applications can be configured from the [Applications page](${manage_url}/#/applications) of the Auth0 dashboard. Third-party applications must be created using the Auth0 Management API and have the `is_first_party` attribute set to false.
+
+## Set JWT token expiration
+
+Set the [ID Token expiration time](/tokens/id-token#token-lifetime) in your Application Settings. By default ID Tokens expire after 10 hours.
+
+Once issued, [an ID Token cannot be revoked](/id-token#revoke-access). So use a short expiration time and renew the session, if the user remains active.
+
+## Do not use wildcards or localhost in callbacks or origins fields
+
+Do not use wildcard or localhost URLs in your application callbacks or allowed origins fields. Using redirect URLs with wildcards [can make your application vulnerable to attacks](https://www.owasp.org/index.php/Unvalidated_Redirects_and_Forwards_Cheat_Sheet).
+
+## Register logout redirect URLs
+
+To redirect users after [logout](/logout), register the redirect URL in your tenant or application settings. Auth0 only redirects to whitelisted URLs after logout.
+
+You should register redirect URLs in your tenant settings. If you need different redirects for each application, you can whitelist the URLs in your application settings.
+
+## Advanced application settings
+
+### RS256 signature algorithm
+
+Make sure that RS256 is the signature method for signing JSON Web Tokens (JWT). The JWT signature method can be found under [Applications > Settings > Advanced Settings > OAuth](${manage_url}/#/applications) on the dashboard.
+ 
+[Auth0 Blog: Navigating RS256 and JWKS](https://auth0.com/blog/navigating-rs256-and-jwks/)
+
+### OIDC compliant
```

Comment:
Should the term used in this heading and in the paragraph below be "OIDC Conformant" since that's what the docs you're linking to use, or is there a reason for it to be "compliant" here (regardless of what we think of the term "OIDC Conformant" in general 😆